README.md

```diff
@@ -7,3 +7,53 @@ The challenges are inspired by the [100 days of code project](https://www.100day
 
 [Телеграм-бот](https://t.me/days_of_code_bot) для проведения марафонов "N days of code" в телеграм-чатах.  
 Марафоны по мотивам проекта [100 дней кода](https://www.100daysofcode.com/)  
+
+## Contributing
+
+We are using [GitHub Issues](https://github.com/lightalloy/days-of-code-bot/issues) for our public bugs. If somebody claims an issue but doesn’t follow up for more than two weeks, it’s fine to take it over but you should still leave a comment.
+
+If you send a pull request, do it against the master branch. For our branches we are using a scheme of (github issue key)-(optional desc/keyword), .e.g. ```34-blah_blah```. We also use this scheme at the beginning of commit message.
+
+## Installation
+
+#### Prerequisites
++ You have Ruby 2.5.5 installed. You can use [rbenv](https://github.com/rbenv/rbenv) to pick a right version for this application.
++ You have installed [bundler](https://bundler.io/)
+
+#### Development Workflow
+
+1) After cloning days-of-code, run ```bundle install``` to fetch its dependencies. 
+
+2) Create your own test bot (see instructions below).
+
+3) Create postgres database, e.g. name it telegram_bot (see example below)
+
+4) Set environment variables DATABASE_URL, BOT_TOKEN and CHAT_ID. You can add them to .bash_profile, e.g.
+```echo 'export DATABASE_URL="postgresql://localhost/telegram_bot?pool=5"' >> ~/.bash_profile```. This will execute it and add the path: ```source ~/.bash_profile```
+
+5) To prevent problems with connection to Telegram API, before bot launch you need to start some preventing online restrictions tool.
+
+You can launch the app in one of two ways:
+1. ```bundle exec ruby bot_control.rb start``` (with console output)
```

Comment:
As I remember `bundle exec ruby bot_control.rb start` is w/o console output, not with, am I right?